packages/camera/src/components/Capture/capture.js

```diff
@@ -64,7 +64,6 @@ const defaultNavigationOptions = {
 };
 
 const Capture = forwardRef(({
-  backgroundColor,
```

Comment:
Hey @natsukiai , I removed this because we already have the colors prop from which we can pull the `colors.background`.